packages/utils/src/ts-eslint/SourceCode.ts

```diff
@@ -208,19 +208,7 @@ declare class TokenStore {
   getTokensBetween<T extends SourceCode.CursorWithCountOptions>(
```

Comment:
```diff
-* @param options The option object. If this is a function then it's `options.filter`.
+* @param options The option object. If this is a number then it's `options.count`. If this is a function then it's `options.filter`.
```

so that it matches the other signatures